services/tmc/src/pages/api/sandbox-results.ts

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable i18next/no-literal-string */
+import axios from "axios"
+import type { NextApiRequest, NextApiResponse } from "next"
+
+import { ClientErrorResponse, GradingResult } from "../../lib"
+
+// Endpoint for the sandbox to report test results
+export default async (
+  req: NextApiRequest,
+  res: NextApiResponse<void | ClientErrorResponse>,
+): Promise<void> => {
+  if (req.method !== "POST") {
+    return res.status(404).json({ message: "Not found" })
+  }
+
+  // verify that the request is coming from sandbox?
```

Comment:
Yes, original request to the sandbox should contain a nonce that will be included when the sandbox posts the result back